src/services/user-service.tsx

```diff
@@ -25,6 +27,24 @@ export const userService = {
       createUrlPath(URLs.users.get, userId, { role: userRole, isEdit })
     )
   },
+
+  getUserByIdWithBaseService: (
+    id: string,
+    userRole: UserRole,
+    isEdit?: boolean
+  ) => {
+    return baseService.request<UserResponse>({
+      method: 'GET',
+      url: getFullUrl({
+        pathname: URLs.users.getUserById,
+        parameters: { id },
+        searchParameters: {
+          role: userRole,
+          isEdit: isEdit !== undefined ? isEdit.toString() : undefined
```

Comment:
I saw the same in one of your previous PRs, did I missed something?
```suggestion
          isEdit: isEdit?.toString() ?? undefined
```

Reply:
yes, I forgot to fix it in this PR too, but I'll first merge the previous PR and then do a rebase

Reply:
You need to rebase your other branch and then do rebase on this branch. Don't use merge, please